tools/python/mwm/ft2osm.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+import sys
+import mwm
+
+if len(sys.argv) < 3:
+    print('Usage: {} <mwm.osm2ft> <ftid>'.format(sys.argv[0]))
+    sys.exit(1)
+
+with open(sys.argv[1], 'rb') as f:
+    osm2ft = mwm.read_osm2ft(f, True)
+
+k = int(sys.argv[2])
+if k in osm2ft:
+    t = {'n': 'node', 'w': 'way', 'r': 'relation'}
```

Comment:
Conversion table for the object type. I use single-character types in the dict to save memory, and here I expand these to create an osm.org url.